test/test-debuglet.ts

```diff
@@ -289,16 +291,26 @@ describe('Debuglet', function() {
   });
 
   describe('setup', function() {
-    before(function() { oldGP = process.env.GCLOUD_PROJECT; });
+    before(function() {
+      oldGP = process.env.GCLOUD_PROJECT;
+    });
 
     after(function() { process.env.GCLOUD_PROJECT = oldGP; });
-
-    beforeEach(function() {
+    let logText = '';
+    let oldLog: any;
+    beforeEach(() => {
       delete process.env.GCLOUD_PROJECT;
+      oldLog = console.log;
```

Comment:
Replacing `console.log` (and the associated restoration code) should no longer be needed, right?